uefi-test-runner/src/boot/misc.rs

```diff
@@ -0,0 +1,10 @@
+use uefi::table::boot::BootServices;
+use core::ptr;
+
+pub fn test(bt: &BootServices) {
+    test_watchdog(bt);
+}
+
+fn test_watchdog(bt: &BootServices) {
+    bt.set_watchdog_timer(0, 0, 0, ptr::null_mut());
```

Comment:
This is a more minor point, but I'm not sure if the UEFI spec allows you to use 0 as a watchdog code. It does say that codes from 0 to 0xffff are reserved by the implementation...

Reply:
Firmware is required to set a 5-minute watchdog timer before running an image, setting the code 0 to 0 disables it.

Reply:
You can find a more detailed explanation here: https://wiki.osdev.org/UEFI#My_UEFI_application_hangs.2Fresets_after_about_5_minutes

Reply:
I opened #27 to resolve this.